package.json

```diff
@@ -39,35 +39,45 @@
   "engines": {
     "node": ">= 14.5.0"
   },
-  "main": "./build/index.js",
-  "types": "./build/index.d.ts",
+  "main": "./build/cjs/index.js",
+  "module": "./build/esm/index.js",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./build/esm/index.d.ts",
+        "default": "./build/esm/index.js"
+      },
+      "require": {
+        "types": "./build/cjs/index.d.ts",
+        "default": "./build/cjs/index.js"
+      }
+    }
+  },
   "files": [
     "./build"
   ],
   "scripts": {
-    "build": "npx tsc --build tsconfig.json",
-    "build:benchmarks": "npx tsc --build ./benchmarks/tsconfig.json",
-    "build:examples": "npx tsc --build ./examples/tsconfig.json",
-    "build:watch": "npx tsc --build --watch tsconfig.json",
```

Comment:
What was the purpose of the watch? If we can check type in watch mode and run tests in watch mode then it's ok